docs/console/company-configuration/providers/extensions/orchestrator-generator.md

```diff
@@ -0,0 +1,270 @@
+| `configMaps`   | [ConfigMount[]](#configmount-schema) | List of configurations to be mounted | |
+| `terminationGracePeriodSeconds` | Number | Grace period time configuration | |
+| `exclusiveServiceExposure` | Boolean | Use this field to understand which containers should be exposed in the k8s Service | ✔️ |
+| `additionalContainers` | [Container[]](#container-schema) | Extra containers to be installed in the service | ✔️ |
+
+#### Env Schema
+
+| Field   | Type     | Description   |
+|---------|----------|---------------|
+| `name`  | String   | Env var name  |
+| `value` | String   | Env var value |
+| `valueType` | `plain`/`secret`  | Env var value (if `secret`, use `secretName` and `secretKey`) |
+| `secretName` | String | Name of the secret holding the env value |
+| `secretKey` | String | Name of the secret key (withint the Secret `secretName`) holding the value for the env |
+
+#### Resource Schema
+
+The `resources` field holds two properties `cpuLimits` and `memoryLimits`, each of them having the following schema:
+
+| Field   | Type     | Description     | Optional |
+|---------|----------|-----------------|----------|
+| `max`   | String   | CPU/MEM limit   | ✔️ |
+| `min`   | String   | CPU/MEM request | ✔️ |
+
+#### Probes Schema
+
+The `probes` field holds two properties `readiness` and `liveness`, each of them having the following schema:
+
+| Field   | Type     | Description     | Optional |
+|---------|----------|-----------------|----------|
+| `path`  | String   | Path to configure the `httpGet` probe   | ✔️ |
+| `port`  | String   | Name of the container port exposing the probe | ✔️ |
+| `initialDelaySeconds` | Number | Value for the `initialDelaySeconds` k8s configuration  | ✔️ | 
+| `periodSeconds` | Number | Value for the `periodSeconds` k8s configuration  | ✔️ | 
+| `timeoutSeconds` | Number | Value for the `timeoutSeconds` k8s configuration  | ✔️ | 
+| `successThreshold` | Number | Value for the `successThreshold` k8s configuration  | ✔️ | 
+| `failureThreshold` | Number | Value for the `failureThreshold` k8s configuration  | ✔️ |
+
+#### ConfigMount Schema
+
+Each `configMaps` item holds information about how to mount a specific ConfigMap found in the top-level `configMaps` field.
+
+Mount information is the following:
+
+| Field   | Type     | Description     | Optional |
+|---------|----------|-----------------|----------|
+| `name` | String | Name of the ConfigMap to be mounted | |
+| `mountPath` | String | Path where the configuration files should be mounted | |
+| `subPaths` | String[] | List of specific files that needs to be picked from the ConfigMap and mounted | ✔️ |
+
+#### Container schema
+
+Each additional container has a similar shape of the main service, holding the following properties subset: `name`, `dockerImage`, `exclusiveServiceExposure`, `containerPorts`, `probes`, `environment`, `resources`, and `configMaps`.
+
+#### Service data model example
```

Comment:
put the example in a collapsed item so that it doesn't clutter the page

Reply:
done